### ISSUE TITLE
Exif data is lost when getting an image from the photo library (iOS)

### ISSUE THREAD

**Issue by ryaa** [apache/cordova-plugin-camera#524]
# Bug Report

## Problem
When opening/adding an image (HEIC or JPEG) from camera roll on iOS and destinationType in camera options is set to DestinationType.FILE_URI the image EXIF data is lost

### What is expected to happen?
Exif data must be preserved.

### What does actually happen?
Exif data is lost.

## Information

The problem exists on iOS only. The camera is used with the below options
```
const options: CameraOptions = {
   destinationType: Camera.DestinationType.FILE_URI,
   sourceType: this.camera.PictureSourceType.PHOTOLIBRARY,
   encodingType: this.camera.EncodingType.JPEG,
   correctOrientation: true
};
```

### Command or Code

N/A

### Environment, Platform, Device

```
ionic info

Ionic:

   Ionic CLI          : 5.2.7 (/Users/alexryltsov/.nvm/versions/node/v10.16.2/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.8
   @ionic/app-scripts : 3.2.4

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 8.0.0, ios 5.0.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.1.3, (and 25 other plugins)

Utility:

   cordova-res : 0.6.0 (update available: 0.8.1)
   native-run  : 0.2.8 (update available: 0.2.9)

System:

   Android SDK Tools : 26.1.1 (/Users/alexryltsov/Library/Android/sdk/)
   ios-sim           : 8.0.1
   NodeJS            : v10.16.2 (/Users/alexryltsov/.nvm/versions/node/v10.16.2/bin/node)
   npm               : 6.9.0
   OS                : macOS Catalina
   Xcode             : Xcode 11.2 Build version 11B52
```

### Version information

cordova-plugin-camera 4.1.0

## Checklist

- [x] I searched for existing GitHub issues
- [x] I updated all Cordova tooling to most recent version
- [x] I included all the necessary information above

**Comment by codeconsole** [comment]
I have created a very easy to run example that demonstrates that EXIF information is lost:
https://github.com/codeconsole/exiftest

@ryaa I am unable to get your version of the plugin working with the latest version of cordova. 
```
% cordova --version
10.0.0
```

```
  git clone https://github.com/codeconsole/exiftest
  cd exiftest
  cordova prepare
  cordova plugin remove cordova-plugin-camera
  cordova plugin add https://github.com/ryaa/cordova-plugin-camera.git
  cordova run ios
  xcrun simctl addmedia booted example_with_exif.jpeg
```

As soon as I try to select a photo, the app crashes (It works fine with the normal plugin)

**Comment by codeconsole** [comment]
Created a pull request with @ryaa 's changes and missing commits from master:
https://github.com/apache/cordova-plugin-camera/pull/712

Can be verified working by running the following:
```
% cordova --version
10.0.0
```

```
  git clone https://github.com/codeconsole/exiftest
  cd exiftest
  cordova prepare
  cordova plugin remove cordova-plugin-camera
  cordova plugin add https://github.com/codeconsole/cordova-plugin-camera.git#ryaa
  cordova run ios
```

Choose a photo and notice all the exif information is there.

**Comment by erisu** [comment]
Resolved by #712
